src/Frontend/Page/FileUpload.hs

```diff
@@ -58,7 +59,43 @@ instance FormPageView BatchCreateUsers where
 theOnlySchoolYearHack :: Int
 theOnlySchoolYearHack = 2016
 
-type CsvUserRecord = [SBS]
+data CsvUserRecord = CsvUserRecord
+    { _csvUserRecordFirst       :: ST
+    , _csvUserRecordLast        :: ST
+    , _csvUserRecordEmail       :: Maybe ST
+    , _csvUserRecordLogin       :: Maybe ST
+    }
```

Comment:
Probably a premature refactoring, but ideally we'd use the newtypes here, as well. And similarly in UserSettingData.